### ISSUE TITLE
Do you change the sigmoid in tower layer to softmax so that the auc of income can't be reduced? 

### ISSUE THREAD

**Issue by tomtang110** [blagojce95/MMOE#1]
Hi

I attempt to change the sigmoid in tower layer to softmax, and the loss is the crossentropy, to my surprise, the auc of income cannot be decreased only 0.54. And the auc of marital is normally increased to 0.96, it's a strange phenomenon, I also tried to reimplement the code in pytorch from [drawbridge/keras-mmoe:](https://github.com/drawbridge/keras-mmoe), I found the same performance. Do you know the reason?

kind regrads
Tang
